CHANGELOG.md

```diff
@@ -2,13 +2,7 @@
 
 ## Unreleased
 ### Added
-- Add getConshdlrName to class Constraint
```

Comment:
Something seems to have been merged wrong here, this line should be in the 4.4.0 release, and your change should be in a new _Unreleased_ section.

Reply:
Oh damn it, I always mess up the CHANGELOG somehow. I think it's alright now. Thanks for looking into this, Mo!